.github/workflows/backend-prod-cd-test.yml

```
@@ -0,0 +1,102 @@
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-24.04
+    outputs:
+      version: ${{ steps.ver.outputs.version }}
+    steps:
+      - uses: actions/checkout@v4
+        with: { fetch-depth: 0 }
+
+      - name: Derive version from PR title (expects like "feat .... (v1.2.3)")
+        id: ver
+        shell: bash
+        run: |
```

actionlint (1.7.8)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)